test/integ/pandas/test_pandas_tools.py

```diff
@@ -602,7 +602,10 @@ def mocked_execute(*args, **kwargs):
             )
             assert m_execute.called and any(
                 map(
-                    lambda e: "CREATE TEMP STAGE" in str(e[0]),
+                    lambda e: (
+                        "CREATE TEMP STAGE" in str(e[0])
+                        or "CREATE SCOPED TEMPORARY STAGE" in str(e[0])
+                    ),
```

Comment:
why it's A or B, I think we should the two condition separately, otherwise we're losing code coverage
use_temp -> CREATE SCOPED TEMPORARY STAGE
not use_temp -> CREATE TEMPORARY STAGE